extensions/wikia/Wall/WallBaseController.class.php

```diff
@@ -100,34 +92,36 @@ public function index( $wallMessagesPerPage = null ) {
 			$wallMessagesPerPage = self::DEFAULT_MESSAGES_PER_PAGE;
 		}
 
-		/* @var Wall wall */
-		$wall = $this->getWallForIndexPage( $title );
-		$wall->setMaxPerPage( $wallMessagesPerPage );
-		$wall->setSorting( $this->getSortingSelected() );
-
-		$threads = $wall->getThreads( $page );
-		$countComments = $wall->getThreadCount();
-
-		$renderUserTalkArchiveAnchor = $this->request->getVal( 'dontRenderUserTalkArchiveAnchor', false ) != true;
+		$this->getThreads( $title, $page, $wallMessagesPerPage );
 
 		$this->response->setVal( 'type', 'Board' );
 		$this->response->setVal( 'showNewMessage', true );
 		$this->response->setVal( 'condenseMessage', true );
 
-		$this->response->setVal( 'wall', $wall );
-		$this->response->setVal( 'threads', $threads );
-		$this->response->setVal( 'renderUserTalkArchiveAnchor', $renderUserTalkArchiveAnchor );
-		$this->response->setVal( 'greeting', $this->getGreetingText( $title ) );
+		$this->response->setVal( 'renderUserTalkArchiveAnchor', $this->request->getVal( 'dontRenderUserTalkArchiveAnchor', false ) != true );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "true" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1125)